tests/unit/test_expression_tree/test_operations/test_evaluate_python.py

```diff
@@ -283,9 +283,9 @@ def test_to_python(self):
         expr = a + b
         constant_str, variable_str = pybamm.to_python(expr)
         expected_str = (
-            "var_[0-9m]+ = y\[0:1\].*\\n"
-            "var_[0-9m]+ = y\[1:2\].*\\n"
-            "var_[0-9m]+ = var_[0-9m]+ \+ var_[0-9m]+"
+            "var_[0-9m]+ = y\\[0:1\\].*\\n"
+            "var_[0-9m]+ = y\\[1:2\\].*\\n"
+            "var_[0-9m]+ = var_[0-9m]+ \\+ var_[0-9m]+"
```

Comment:
Can these be made into raw strings instead of manual escapes or does that break the regex?

Reply:
Yes we can convert all of them to raw string without any breaking change.

Reply:
Ok then I would go ahead and do it the raw string method